### ISSUE TITLE
JSON Error and no such file or directory

### ISSUE THREAD

**Issue by wass3r** [zwilias/elm-coverage#23]
I'm having an issue when trying to run `npx elm-coverage -v src/elm` in my Elm 0.19.1 project. The output ends with:
```bash
[12:44:40.01] Ruh roh, tests failed.
[12:44:40.01] Something went wrong: 
Error: Received invalid JSON from test interface search: SyntaxError: Unexpected token e in JSON at position 0
```

If I use the `-f` flag, I get:

```bash
[12:31:19.08] Something went wrong: 
Error: ENOENT: no such file or directory, scandir '.coverage/instrumented/elm-stuff/generated-code/elm-explorations/test'
```

I have this block in my `elm.json`, if it matters:

```json
"test-dependencies": {
    "direct": {
        "elm-explorations/test": "1.2.1"
    },
    "indirect": {}
}
```

Any ideas? Running `elm-test` on its own works fine.

Elm: 0.19.1
Elm-Test: 0.19.1
NodeJS: 10.15.3

**Comment by gyzerok** [comment]
Have the same problem using 0.19.1.

**Comment by zwilias** [comment]
Fixed in https://github.com/zwilias/elm-coverage/commit/d594f6c4f916076ea3c2deea7cabb50348d1d11b and released as 0.3.0! 🚀
